app/views/categories/index.html.erb

```diff
@@ -0,0 +1,50 @@
+<body>
+  <main>
+    <h2>Products by Category</h2>
+
+  <section class= "product-list">
+  <h4>Annual</h4>  
+    <% @product.categories each do |annual| %>   
+    <h4><%= product.name.capitalize %></h4>
+      <ul>
+        <li>Description: <%= product.description.capitalize %></li>
+        <li>Price: $<%= product.price %></li>
+        <li>Stock: <%= product.stock %> units</li>
+        <li>Available: <%= product.available %></li>
+        <li>User_id: <%= product.user_id %></li>
+        <li>photo_url</li>
```

Comment:
Does this need to be wrapped in an image tag somehow and reference 'product.photo_url'?

Reply:
This is just a first draft, I'm updating the seeds and will update that to a link :)